neurolib/optimize/evolution/evolution.py

```diff
@@ -700,34 +802,113 @@ def loadEvolution(self, fname):
         :rtype: self
         """
         import dill
+
         evolution = dill.load(open(fname, "rb"))
-        evolution.__init__(lambda x: x, self.parameterSpace)
+
+        # parameter space is not saved correctly in dill, don't know why
```

Comment:
this is weird...do you have any idea why? what's the problem when you load with `dill`?